r/R/differential_expression.R

```diff
@@ -55,13 +54,19 @@ runDE <- function(req, data){
     message("checking meta.data slot ", str(data@meta.data))
 
     # Compute differential expression
-    result <- FindMarkers(data, group.by = "custom", ident.1 = "base", ident.2 = "background")
+    result <- presto::wilcoxauc(data, assay = "data", seurat_assay = "RNA",group_by="custom")
+    result <- result[result$group=="base",]
+    rownames(result) <- result$feature
+    result <- result[,c("pval","logFC","pct_in","pct_out","padj","auc")]
```

Comment:
Renaming the columns to maintain compatibility with the UI and eventually Seurat if we want to add it as alternative method (Just add 1 line and we have seurat)